lib/dor/datastreams/workflow_ds.rb

```diff
@@ -24,7 +23,7 @@ def get_workflow(wf, repo = 'dor')
       Workflow::Document.new(xml.to_s)
     end
 
-    alias_method :[], :get_workflow
+    alias :[] :get_workflow
```

Comment:
I believe @atz used `alias`, as suggested by the ruby style guide + rubocop:

https://github.com/bbatsov/ruby-style-guide#alias-method-lexically